adopters.md

```diff
@@ -0,0 +1,107 @@
+Who is using Flatcar?
+====================
+
+The following is a list of adopters who have publicly spoken about their use of Flatcar, or who have added themselves to this list.
+
+
+Adding yourself as a user
+-------------------------
+
+If you are using Flatcar, please consider adding yourself as a user with a quick description of your use case by opening a pull request to this file and adding a section describing your usage of Flatcar. If you are open to others contacting you about your use of Flatcar on Slack or Matrix, include your Slack/Matrix nickname or email as well.
+    
+    * N: Name of user (company or individual) 
+      D: Description 
+      L: Link with further information (optional) 
+      C: Contacts available for questions (optional)
+
+Example entry:
+
+    * N: Flatcar Example User Inc.
+      D: Using Flatcar for running Kubernetes in Azure via Cluster API
+      L: https://www.exampleuser.com 
+      C: Slack: @slacknick and/or Matrix: @Matrixnick and/or Email: nick [at] exampleuser [dot] com
+    
+
+Requirements to be listed
+-------------------------
+
+ * You must represent the user listed. Do not add entries on behalf of
+   other users.
+* Commercial or production use is not required. A user can be an end user, cloud service provider or consultant as long as it is a permanent deployment and not a trial deployment. A well-done home lab setup can be equally
+   interesting as a large-scale commercial deployment.
+
+
+Users 
+-----
+ 
+    * N: Adobe
+    
+    * N: AloPeyk
+      D: We are an on-demand delivery business that every day serve millions of requests on a consistent bare metal infrastructure have this concern to choose a reliable and atomic operations system but none of the common OS couldn't satisfy us for such heavy workloads. Since we migrated our production Kubernetes cluster nothing can break this consistent and solid cluster which is powered by amazing Flatcoar OS.
+    
+    * N: Cloud house
+      D: Flatcar OS has been useful for us for our on-premise solution to our customers 
+      
+    * N: Equinix Metal
+L: https://kinvolk.io/blog/2021/02/case-study-equinix-metal-builds-on-flatcar/
+D: Equinix uses Flatcar as the OS for its bare metal cloud control plane, which runs in Kubernetes
+      
+    * N: Finleap Connect
+      D: Finleap Connect - At finleap connect we serve over a million financial transactions per day. As a regulated company using a pure cloud-native stack based on Kubernetes, using Flatcar as our foundational building block for reliable, secure and immutable nodes across the public clouds and on bare-metal deployments was a day one decision we never regretted. Today Flatcar serves all of our 12 production clusters with over 300 nodes on public clouds in 3 countries and on our bare-metal private-cloud setup.
+    
+    * N: Host with quantum 
+      C: https://twitter.com/klimpong
```

Comment:
```suggestion
      L: https://www.planetary-quantum.com/
      D: Planetary Quantum is a berlin-based provider of Docker-hosting and application hosting. Our sister company Planetary Networks colocates their private cloud in two (fiber-)interconnected datacenters in Berlin and Quantum offers container-based solutions (Docker Swarm and a custom tailored application hosting) on top of Flatcar Linux. Flatcar Linux is a great choice for us because it's a modern Linux, well-suited for Docker and Kubernetes due to recent versions of Kernel, SystemD, immutable root and a well-tested userland. Simple and straight-forward updates of the OS make running Flatcar a no-brainer for us. We currently operate over 50 clusters for our customers in our private cloud — all based on Flatcar Linux.
```